src/sentry/integrations/github/client.py

```diff
@@ -45,21 +45,24 @@
 
 class GithubRateLimitInfo:
     def __init__(self, info: dict[str, int]) -> None:
-        self.limit = info["limit"]
-        self.remaining = info["remaining"]
-        self.reset = info["reset"]
-        self.used = info["used"]
+        self.limit = info.get("limit")
+        self.remaining = info.get("remaining", 0)
+        self.reset = info.get("reset")
+        self.used = info.get("used")
```

Comment:
would rather not weaken these -- the brackets are preferrable since we know the structure (whereas `.get` would allow a programming error to slip through unnoticed on a typo)